### DISCUSSION TITLE
Feature Request: Ability to pass target directly to Link and avoid using <a> tags.

### DISCUSSION THREAD

**Discussion by lingdocs** [vercel/next.js#33566] [Ideas]
### Describe the feature you'd like to request

One of the great things about using the `next/link` is that you can avoid the painful ordeal of removing all styling from anchor elements if you just want a plain link, without any formatting.

```jsx
<Link to="https://www.example.com">
    <h3>Example Link w/ No Special Anchor Styling 🙂</h3>
</Link>
```

But if I wanted my external link to open in a new tab, [I need to specify the target in an anchor element](https://github.com/vercel/next.js/discussions/15486#discussioncomment-42331). Now I have to deal with removing any anchor styles, which can be a nightmare depending on your CSS framework setup.

```jsx
<Link to="https://www.example.com">
    <a target="_blank">
        <h3>Now I have anchor styling to remove! 😩</h3>
    </a>
</Link>
```

### Describe the solution you'd like

To avoid having to use the `<a>` tag cases like this I would like **to be able to pass in a `target` and `rel` directly to the `Link` component**.

```jsx
<Link to="https://www.example.com" target="_blank" rel="noreferrer">
    <h3>Example Link w/ No Special Anchor Styling 🙂</h3>
</Link>
```

### Describe alternatives you've considered

My only option now is to create a class to unstyle certain `<a>` elements [which is a pain with Bootstrap](https://stackoverflow.com/questions/22429881/how-to-unstyle-anchor-when-using-bootstrap).

**Comment by CyberCyclone** [comment]
@balazsorban44 I'm not sure why you converted this into a discussion when this is clearly a missing function that should be there in Next.js by default. The most obvious use case is opening a link to a different website in a new tab so parent website doesn't get replaced in the current tab.

**Reply by balazsorban44** [reply]
Hi, the Feature request issue form states:

> Feature requests will be converted to the GitHub Discussions "Ideas" section.

We focus the issue tracker on bugs and look into requests after they have gained some traction by the community. This helps us prioritize the problems we should focus on.
